doc/user_manual/chSpecialElements.tex

```diff
@@ -1652,23 +1652,290 @@ \section{Tracking Code for Collimation Studies} \label{sec:collimat}
 The introduction of the optic parameter $\beta$ allows studies of error scenarios (orbit distortion, beta-beating\index{beta-beating}) and/or to see the effect of misaligned collimators.
 This structure is then repeated within the file for each of the collimators to be included in the study.
 
+% ================================================================================================================================ %
+
 \subsection{Collimation Input Block} \label{sec:coll:input}
 
-The collimation module is controlled by the \texttt{COLL} block in \texttt{firt.3}.
+The collimation module is controlled by the \texttt{COLL} block in \texttt{fort.3}.
 
 \bigskip
 \begin{tabular}{@{}lp{0.8\linewidth}}
     \textbf{Keyword}    & \texttt{COLL}\index{COLL} \\
-    \textbf{Data lines} & 17\\
+    \textbf{Data lines} & Either 17 lines (old format), or free (new format)\\
     \textbf{Format}     & See below.
 \end{tabular}
 
 \paragraph{Format Description:}~\\
 
+From SixTrack version 5.2.3 and on, the format of the \texttt(COLL) block is a set of keyword/value lines.
+This format is described in Section~\ref(sec:coll:newfmt).
+
+For backwards compatibility, the old format is still supported, and it is described in Section~\ref{sec:coll:oldfmt}..
+When encountering the \texttt{COLL} block, the parser will check if the first item of the line is a valid keywords, and if it is not, it will fall back to parsing the block in the old format.
+The old block format consists of 17 lines of varying number of values, and the full description is laid out in Table~\ref{tab:COLL_INP}.
+
+Note that the old and new format cannot be inter-mixed, as the old format is parsed based on the line number.
+The 17 lines must follow exactly the format described in Table~\ref{tab:COLL_INP}.
+
```

Comment:
Sorry for necromancing the thread (and not just RTFC'ing), but: Regarding the DO_COLL and the new format, it is written that if DO_COLL is false, the memory is not allocated.

Does this imply that DO_COLL must be on the first line of the new collimation block, so that the memory used for parsing is actually allocated correctly before parsing the rest?

Reply:
No. The collimation tracking arrays have nothing to do with input parsing.